styles/components/BlockImageText.scss

```diff
@@ -1,4 +1,8 @@
 .BlockImageText {
+  &__description.small > div > span > p {
+    line-height: 1.5rem;
```

Comment:
Looking at this stylesheet, I don't understand why we need this since we have line 8.. which means, there is somewhere else that is overriding this style. Where is it being overridden?

Reply:
Oh it's because I realized that i didn't delete the line 8 after I made the screenshot above to share with you. The line 8 should go to fix the line-height issue for big/serif text in this block. I removed it and push the commit but seemed like you reviewed this while i did that @mokaymokay - if you see the latest commit, it is removed!